pkg/client/client_test.go

```diff
@@ -0,0 +1,52 @@
+package client
+
+import (
+	"errors"
+	"testing"
+)
+
+func Test_validateAndSanitizePrismCentralInfoAddress(t *testing.T) {
+	tests := []struct {
+		name            string
+		providedAddress string
+		expectedAddress string
+		expectedErr     error
+	}{
+		{
+			"with scheme",
+			"https://prism-bowser.ntnxsherlock.com",
+			"prism-bowser.ntnxsherlock.com",
```

Comment:
pls add one test with ip address as well

Reply:
added some tests to address this